src/style.css

```
@@ -0,0 +1,117 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: bold;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-size: 16px;
+}
+
+.big_search_bar input {
+  --height-big-search-bar: 70px;
+  width: 100%;
+  height: var(--height-big-search-bar);
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  
+}
+
+.big_search_bar input::placeholder {
+  color: #3d4e61;
+}
+
+.big_search_bar .container {
+  position: relative;
+}
+
+.big_search_bar img {
+  position: absolute;
+  left: 28px;
+  top: 25px;
+}
+
+.small_search_bar,
+.big_search_bar {
+  margin-top: 20px;
+}
+
+.big_search_bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small_search_bar input {
+  --height-small-search-bar: 42px;
+  width: 100%;
+  height: var(--height-small-search-bar);
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+}
+
+.small_search_bar::placeholder {
+  color: #3d4e61;
+}
+
+.big_search_bar input:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+  outline: 0;
+  font-weight: 900;
+}
+
+.small_search_bar .container {
+  position: relative;
+}
+
+.small_search_bar img {
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
+  width: 11px;
+  height: 11px;
+}
+
+.small_search_bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.small_search_bar input:focus-within {
```

```suggestion
.small_search_bar input:focus {
```

Не используй составные селекторы